ansible/roles/ocp-workload-migration/README.adoc

```diff
@@ -29,28 +29,7 @@ ansible-playbook -i "bastion.${GUID}.${BASE_DOMAIN}", ./ansible/configs/ocp-work
 
 === Variables
 
-This workload does not require any variables. Following variables can be overridden if you do not want default behaviour.
-
-|===
-| Variable | Default Value | Description
-
-| mig_operator_repo
-| https://github.com/fusor/mig-operator
-| Repository for migration operator
-
-| mig_operator_repo_branch
-| master
-| Branch in operator repo
-
-| mig_operator_version
-| latest
-| Migration operator version 
-
-| mig_ocp_version
-| 4
-| Version of OpenShift cluster
-|===
-
+This workload does have require any variables. 
```

Comment:
Should this read "does not have"?

Reply:
but wait... isn't `mig_migration_namespace` a variable for this role?

Reply:
@jkupferer we decided to make `mig_migration_namespace` a configurable variable for expected future use-cases, but we don't actually want the user to modify this variable for now (changing the namespace will break things badly until we make changes in our codebase).